packages/root-config/src/assets/plugin.importmap.json

```diff
@@ -14,6 +14,6 @@
     "@zuri/zuri-plugin-tools": "https://externaltools.zuri.chat/zuri-zuri-plugin-tools.js",
     "@zuri/zuri-plugin-channels": "https://channels.zuri.chat/static/main/dist/zuri-zuri-plugin-channels.js",
     "@zuri/zuri-plugin-contributions": "https://ct.zuri.chat/js/app.js",
-    "@zuri/zuri-plugin-messaging": "https://chat.zuri.chat/zuri-messaging.js"
+    "@zuri/zuri-plugin-messaging": "http://localhost:5013/zuri-messaging.js"
```

Comment:
When you finish coding, you should change it back to how its supposed to be

Reply:
Alright. I will correct it.